packages/router/src/index.ts

```diff
@@ -25,4 +25,19 @@ export { usePageLoadingContext } from './page-loader'
 export { default as RouteAnnouncement } from './route-announcement'
 export * from './route-announcement'
 
+/**
+ * A more specific interface will be generated by
+ * babel-plugin-redwood-routes-auto-loader when a redwood project is built
+ *
+ * @example
+ * interface AvailableRoutes {
+ *   home: () => "/"
+ *   test: () => "/test"
```

Comment:
Could you include an example here with your new cool params typings?

Reply:
Sure